jar-infer/test-java-lib-jarinfer/build.gradle

```diff
@@ -18,6 +18,8 @@ plugins {
   id "java"
 }
 
+evaluationDependsOn(":jar-infer:jar-infer-cli")
```

Comment:
@kageiit Is this okay to be here?
Otherwise Gradle complains: `only buildscript {} and other plugins {} script blocks are allowed before plugins {} blocks, no other statements are allowed`

Reply:
Its ok to be here